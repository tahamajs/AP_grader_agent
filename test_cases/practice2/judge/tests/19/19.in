enter 1
reserve_table 1 1
enter 2
reserve_table 2
show_table_info 1
reserve_table 3 2
reserve_table 4
enter 5
reserve_table 5 2
reserve_table 7
show_table_info 1
reserve_table 8 1
reserve_table 9
show_table_info 1
reserve_table 10 5
reserve_table 11
exit 1
show_table_info 1
reserve_table 12
switch 4
exit 2
show_table_info 1
reserve_table 14 1
exit 4
show_table_info 1
reserve_table 15
reserve_table 16 6
reserve_table 17
show_table_info 2
exit 7
show_table_info 1
reserve_table 18 1
reserve_table 19
show_table_info 2
reserve_table 20 2
reserve_table 21
reserve_table 22 5
switch 3
show_table_info 1
show_table_info 2
reserve_table 23 3
reserve_table 24
reserve_table 25 4
reserve_table 26
exit 10
show_table_info 5
reserve_table 27
reserve_table 28 6
reserve_table 29
show_table_info 6
