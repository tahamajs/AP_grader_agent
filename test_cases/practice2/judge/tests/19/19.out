Table 1: 4 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Paria sits at table 1
Table 1: 3 0
Table 2: 4 0
Table 5: 3 0
Table 6: 3 0
Table 3: 2 0
Table 4: 2 0
Amir Hossein sits at table 1
Table ID: 1
People at the table: Amir Hossein, Paria
Table remaining capacity: 2
Waiting queue length: 0
Parsa sits at table 2
Mehrad sits at table 1
Table 1: 1 0
Table 2: 3 0
Table 5: 3 0
Table 3: 2 0
Table 6: 3 0
Table 4: 2 0
Fereshte sits at table 2
Mahdi sits at table 1
Table ID: 1
People at the table: Amir Hossein, Mahdi, Mehrad, Paria
Table remaining capacity: 0
Waiting queue length: 0
Taha enters the waiting queue of table 1
Sadegh enters the waiting queue of table 1
Table ID: 1
People at the table: Amir Hossein, Mahdi, Mehrad, Paria
Table remaining capacity: 0
Waiting queue length: 2
Hamid sits at table 5
Shaghayegh enters the waiting queue of table 1
Paria exits!
Table ID: 1
People at the table: Amir Hossein, Mahdi, Mehrad, Taha
Table remaining capacity: 0
Waiting queue length: 2
Bob enters the waiting queue of table 1
Mehrad switches seats with Taha!
Amir Hossein exits!
Table ID: 1
People at the table: Mahdi, Mehrad, Sadegh, Taha
Table remaining capacity: 0
Waiting queue length: 2
Babak enters the waiting queue of table 1
Mehrad exits!
Table ID: 1
People at the table: Mahdi, Sadegh, Shaghayegh, Taha
Table remaining capacity: 0
Waiting queue length: 2
Erfan enters the waiting queue of table 1
Alex sits at table 6
Sipamo enters the waiting queue of table 1
Table ID: 2
People at the table: Fereshte, Parsa
Table remaining capacity: 2
Waiting queue length: 0
Mahdi exits!
Table ID: 1
People at the table: Bob, Sadegh, Shaghayegh, Taha
Table remaining capacity: 0
Waiting queue length: 3
Amin enters the waiting queue of table 1
Sina sits at table 2
Table ID: 2
People at the table: Fereshte, Parsa, Sina
Table remaining capacity: 1
Waiting queue length: 0
Reza sits at table 2
Tommy enters the waiting queue of table 1
Kiana sits at table 5
Parsa switches seats with Sadegh!
Table ID: 1
People at the table: Bob, Parsa, Shaghayegh, Taha
Table remaining capacity: 0
Waiting queue length: 5
Table ID: 2
People at the table: Fereshte, Reza, Sadegh, Sina
Table remaining capacity: 0
Waiting queue length: 0
Narges sits at table 3
Sara enters the waiting queue of table 1
Saman sits at table 4
Walter enters the waiting queue of table 1
Hamid exits!
Table ID: 5
People at the table: Kiana
Table remaining capacity: 2
Waiting queue length: 0
Jesse enters the waiting queue of table 1
Hank sits at table 6
Gustavo enters the waiting queue of table 1
Table ID: 6
People at the table: Alex, Hank
Table remaining capacity: 1
Waiting queue length: 0
